rest/route/github.go

```diff
@@ -280,6 +281,20 @@ func (gh *githubHookApi) Run(ctx context.Context) gimlet.Responder {
 			}))
 			return gimlet.MakeJSONErrorResponder(errors.Wrap(err, "can't enqueue item on commit queue"))
 		}
+
+		pushJob := units.NewGithubStatusUpdateJobForPushToCommitQueue(userRepo.Owner, userRepo.Repo, baseBranch, PRNum)
+		pushJob.Run(ctx)
+		grip.ErrorWhen(pushJob.Error() != nil, message.WrapError(pushJob.Error(), message.Fields{
```

Comment:
you don't need to do error when, just pass a message.WrapError() message and its only loggable if error isn't nil.

Reply:
Changed this to grip.Error.